autoscheduler/scraper/tests/commands/scraper_utils_tests.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from datetime import datetime
+from scraper.management.commands.utils.scraper_utils import (
+    get_recent_terms, get_recent_semesters,
+)
+
+class GetRecentSemestersTests(unittest.TestCase):
+    """ Tests for get_recent_semesters """
+
+    def setUp(self):
+        self.strp_format = "%m/%d/%Y"
+        self.spring_2021 = ['20211']
+        self.summer_fall_2020 = ['20202', '20203']
+        self.summer_fall_2021 = ['20212', '20213']
+
+    def test_dec30_2020_is_spring_2021(self):
+        """ Tests that 11/30/2020 results in spring 2021 (20211) """
+        date = datetime.strptime('12/30/2020', self.strp_format)

```

Comment:
Up to you if you want to change it but I think using `date = datetime(year, month, day)` for these dates looks cleaner, and avoids having to specify the format

Reply:
Yeah you're totally right I like this a lot better, will change it to this